index.src.html

```diff
@@ -148,26 +148,6 @@ <h2 id="terms">Key Concepts and Terminology</h2>
       a decade, the practical risk of confusion seems low.
     </dd>
 
-    <dt>
-      <dfn export local-lt="a priori authenticated" oldids="a-priori-insecure-url a-priori-insecure-origin">
```

Comment:
We should retain this ID somewhere, given that's it's somewhat widely used. Perhaps a note (`<p class="note">`) that contains some ID'd text noting that the old term is now the new term?

Reply:
Good point, I added a note below (at the end of the definitions section). Thanks for reviewing.